org.andrew.ytdownloader.yml

```diff
@@ -0,0 +1,53 @@
+app-id: org.andrew.ytdownloader
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=home
```

Comment:
Does it really need full home access?

Reply:
Can the user save files to his home directory if this permission is not given?

Reply:
how about `xdg-download`?

Reply:
The user can select default download location in preferences. So don't I need full home access?